### PR TITLE
Make reference_edit_text_hint translatable

### LINKED ISSUE #473: Make reference_edit_text_hint translatable
Mark `reference_edit_text_hint` on repeating groups as a translatable field to allow JMAG placeholder substitution.

### PR DESCRIPTION
**Pull request by syedowaisali90**
Hi, @vincent-karuri I have created a PR for review.
Close #473

### CODE REVIEW

**Comment by vincent-karuri** [comment]
After the build resolves, please re-generate assets against the diagnostics form using JMAG to allow translation of the `edit_text_hint` field.

File: rdt-open-reader/src/covidIndonesia/assets/json.form/patient-diagnostics-form.json [hunk 1]
```diff
@@ -948,7 +948,7 @@
           {
             "key": "covid_patient_in_compound_name_label",
             "type": "label",
-            "text": "Contact's name",
+            "text": "Contact\u0027s name",
```
Comment (vincent-karuri) [inline rdt-open-reader/src/covidIndonesia/assets/json.form/patient-diagnostics-form.json:951]: Any reason this is changing to unicode?

Comment (syedowaisali90) [reply rdt-open-reader/src/covidIndonesia/assets/json.form/patient-diagnostics-form.json:951]: i just generate assets from MLS

Comment (vincent-karuri) [reply rdt-open-reader/src/covidIndonesia/assets/json.form/patient-diagnostics-form.json:951]: ok

**Review by vincent-karuri — APPROVED** [review]

**Pull request merged**